### DISCUSSION TITLE
How is Directus built and the code structured?

### DISCUSSION THREAD

**Discussion by biapar** [directus/directus#7826] [Archive]
How is Directus built and the code structured?
Which modules use it?
Thanks.

**Comment by rijkvanzanten** [comment]
This is a little too broad of a question for me to answer accurately.. I'm not sure what the exact information is you're looking for. That being said, the whole code base is open source, so literally all the information you could possibly need on that front should be available right in this here repo 🙂

**Reply by biapar** [reply]
How is built and structured?

**Reply by rijkvanzanten** [reply]
Apologies, but I just don't understand what sort of information you're looking for. If it's folder/file structure, it's available in this repo. Otherwise you'll have to be a bit more explicit in what information you're looking for
